### PR TITLE
Added english SDK Translation

### PR DESCRIPTION
**Pull request by a1300**
Dear Asch-Team,

I added the english translation for the SDK documentation.

Please merge first the [105 Pull Request](https://github.com/AschPlatform/asch/pull/105). :-)

Thank you!
Kind regards
a1300

### CODE REVIEW

File: docs/asch_sdk_api_en.md [hunk 1]
```diff
@@ -0,0 +1,622 @@
...
+## **1. Database**
+
+### 1.1 `aync` app.sdb.load(model, fields, indices)
+
+- `model` Datamodel
+- `fields` Fields of the data model
+- `indices` Array of indexes, it can be a single field index or a multi field index (provide an string array)
+
+> The operation has no return value, it throws an Exception when an error occurs
+> 将指定模型的数据加载到内存并建立索引, 这样可以提高查询和更新一个状态的效率
```
Comment (yanyiwu) [inline docs/asch_sdk_api_en.md]: why Chinese here?

File: docs/asch_sdk_api_en.md [hunk 2]
```diff
@@ -0,0 +1,622 @@
...
+app.validate('amount', '1e10') // throws
+```
+
+### 8.2 app.registerContract(type, name)
+
+- `type` Contract Type
+- `name` The name of contract
+
+> No return value
+> 为合约注册一个数字类型, 未注册的合约无法被外部调用
```
Comment (yanyiwu) [inline docs/asch_sdk_api_en.md]: Why Chinese here?

**Comment by yanyiwu** [comment]
Merged, but it has some Chinese words in your english document pull request, maybe you can delete them later.

File: docs/asch_sdk_api_en.md (hunk 2 above)
Comment (a1300) [reply docs/asch_sdk_api_en.md]: I couldn't translate it in a meaningful way.

File: docs/asch_sdk_api_en.md (hunk 1 above)
Comment (a1300) [reply docs/asch_sdk_api_en.md]: I couldn't translate it in a meaningful way.

**Pull request merged**